coverage/pom.xml

```diff
@@ -14,46 +14,46 @@
 
 	<name>coverage</name>
 	<dependencies>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-core</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-vlog</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-rdf</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-owlapi</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-graal</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-parser</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
-          <dependency>
-                <groupId>${project.groupId}</groupId>
-                <artifactId>rulewerk-commands</artifactId>
-                <version>${project.version}</version>
-          </dependency>
-	  <dependency>
-		<groupId>${project.groupId}</groupId>
-		<artifactId>rulewerk-client</artifactId>
-		<version>${project.version}</version>
-	  </dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-core</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-vlog</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-rdf</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-owlapi</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-graal</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-parser</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-commands</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rulewerk-client</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
This needs a dependency on `rulewerk-reliances` as well, cf. the comment in the main `pom.xml`; otherwise the coverage aggregation will not work correctly.

Reply:
Thank you for the review. It is added now in a new commit